shared-libs/transitions/src/transitions/update_clinics.js

```diff
@@ -8,6 +8,10 @@ const FACILITY_NOT_FOUND = 'sys.facility_not_found';
 
 const config = require('../config');
 
+const getConfig = () => config.get(NAME) || [];
+
+const getConfiguredForm = (form) => form && getConfig().find(item => item && item.form === form);
```

Comment:
I don't think you need to null-check `form` here, right? You're only calling this function after you test it's defined.

Reply:
You're right. CHnaging it.